doc/source/ray-contribute/development.rst

```diff
@@ -263,11 +263,11 @@ You can also use the included script to install Bazel:
    3. Go to "For Developers" on the left pane;
    4. Turn on "Developer mode".
 
-2. Add the following Miniconda subdirectories to PATH. If Miniconda was installed for all users, the following paths are correct. If Miniconda is installed for a single user, adjust the paths accordingly.
+2. Add the following Miniforge subdirectories to PATH. If Miniforge was installed for all users, the following paths are correct. If Miniforge is installed for a single user, adjust the paths accordingly.
 
-   - ``C:\ProgramData\Miniconda3``
-   - ``C:\ProgramData\Miniconda3\Scripts``
-   - ``C:\ProgramData\Miniconda3\Library\bin``
+   - ``C:\ProgramData\Miniforge3``
+   - ``C:\ProgramData\Miniforge3\Scripts``
+   - ``C:\ProgramData\Miniforge3\Library\bin``
```

Comment:
I hope these are correct replacements. I have never used `Miniforge 3` on Windows. Do you want me check it out on Windows?